sympy/geometry/point.py

```diff
@@ -172,28 +176,31 @@ def is_collinear(*args):
         False
 
         """
+        args = [Point.pointify(p) for p in args]
 
         # Coincident points are irrelevant; use only unique points.
-        args = list(set(args))
-        if not all(isinstance(p, Point) for p in args):
-            raise TypeError('Must pass only Point objects')
+        # XXX: the current workaround for issue #11238 relies on the points
+        # being in a specific order, so don't let `set` change the order
+        #args = list(set(args))
```

Comment:
I think #11554 does fix this.  I tried to update the docstring before and got really frustrated with sphinx errors.  I'll try again.